nrf_modem/doc/dectphy.rst

```diff
@@ -0,0 +1,299 @@
+.. _nrf_modem_dect_phy:
+
+DECT NR+ physical layer
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The DECT NR+ physical layer (PHY) interface in the Modem library is used to control the DECT NR+ PHY in the DECT NR+ modem firmware.
+
+The DECT NR+ modem firmware is a variant of the nRF9161 modem firmware with a different feature set than the cellular modem firmware.
+In particular, the DECT NR+ modem firmware does not support cellular operation or the Global Navigation Satellite System (GNSS).
+
+For details about the key hardware capabilities of the nRF9161 and its DECT NR+ PHY modem firmware implementation, with regards to the DECT NR+ standard, refer to the `nRF9161 DECT NR+ product specification`_.
+
+.. note::
+   To obtain the DECT NR+ modem firmware, you must contact the Nordic Semiconductor sales department.
+
+For more details about which Modem library features are available for which modem firmware, see :ref:`nrf_modem_flavour`.
+
+Regulations
+***********
+
+DECT NR+ operates on free but regulated radio channels. The availability of the channels and the exact regulation to use them varies in different countries.
+
+.. note::
+   It is your responsibility to operate the devices according to the local regulation, in all situations.
+
+.. important::
+   The European Telecommunications Standards Institute (ETSI) has published European Normative document `ETSI EN 301 406-2`_, which restricts operation on specific absolute channel frequency numbers within the available range.
+   If you plan to operate a device using the DECT NR+ PHY modem firmware in territories where ETSI EN 301 406-2 is applicable, you must refer to table 4.3.2.3-2 to see which channel frequency numbers you are allowed to use.
+
+   The Federal Communication Commission (FCC) rules and regulations for radio frequency devices are located in `47 CFR Part 15`_.
+   If you plan to operate a device using the DECT NR+ PHY modem firmware in territories where 47 CFR Part 15 is applicable, you must refer to `47 CFR Part 15.323`_ to see which channel frequency numbers you are allowed to use.
+
+Protocol stack overview
+***********************
+
+The physical layer is only one part of the DECT-2020 new radio protocol stack (DECT NR+).
```

Comment:
The name for this technology is `DECT-2020 New Radio`, `New Radio` is part of the name and it is capitialized.

Reply:
Agree.